src/palace/manager/api/admin/controller/view.py

```diff
@@ -67,8 +67,8 @@ def __call__(self, collection, book, path=None):
         admin_js = AdminClientConfig.lookup_asset_url(key="admin_js")
         admin_css = AdminClientConfig.lookup_asset_url(key="admin_css")
 
-        # We always have local_analytics
-        show_circ_events_download = True
+        # # We always have local_analytics
+        # show_circ_events_download = True
```

Comment:
Minor: Should be able to remove this comment.

Reply:
There were two things I was trying to accomplish:
- Have one home for configuration of feature flags; and
- Avoid accidentally overriding configuration that we have decided is fixed (see the hard-coded values I pulled back into the feature flags object).

Being able to prevent override by the environment variables preserves the existing behavior of those pre-existing feature flags (i.e., they could not be overridden by the environment -- they were fixed).

I believe the main use case for this is to avoid accidentally reconfiguring some functionality that we have decided is now 'fixed' into the system. I think should mostly apply only to functionality that is in transition. That said, we haven't gone back to the admin UI and removed these fixed feature flags. I'm not sure if that's because we want them there or because we just haven't done anything about them.

I made a separate class for this primarily to make updates to the feature flags a littel clearer and less cluttered.

Reply:
I'm fine with this going in as is, so feel free either way. 

My opinion on this though is that we should allow the feature flags to be overridden by env vars and any of the flags we want to be set as fixed, we can put in tickets to remove the flag and clean up the config in the admin ui and CM.